### PR TITLE
feat: updated library from protos

### PR DESCRIPTION
**Pull request by telpirion**
This PR does the following:

- Updates the library to use Microgenerators
- Updates the library to include v1beta2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Google APIs Client Libraries, in [Client Libraries Explained][explained].
 * [Quickstart](#quickstart)
   * [Before you begin](#before-you-begin)
   * [Installing the client library](#installing-the-client-library)
-  * [Using the client library](#using-the-client-library)
```
Comment (JustinBeckwith) [inline README.md]: This reads like the quickstart got deleted .... (will keep reading)

File: synth.py [hunk 2]
```diff
@@ -21,17 +21,21 @@
 logging.basicConfig(level=logging.DEBUG)
 
 # run the gapic generator
-gapic = gcp.GAPICGenerator()
-versions = ['v1beta1']
+gapic = gcp.GAPICMicrogenerator()
+versions = ['v1beta1','v1beta2']
```
Comment (JustinBeckwith) [inline synth.py]: Do we need to ship both `v1beta1` and `v1beta2`?  Could we just ship 2 and keep it simple?

File: synth.py [hunk 3]
```diff
@@ -21,17 +21,21 @@
...
+   version=version)
  s.copy(library, excludes=['README.md', 'package.json'])
 
 # Copy common templates
 common_templates = gcp.CommonTemplates()
-templates = common_templates.node_library()
+templates = common_templates.node_library(source_location='build/src')
 s.copy(templates, excludes=[])
 
 # Node.js specific cleanup
```
Comment (JustinBeckwith) [inline synth.py]: There is a `npx compileProtos` step or something in the other `synth.py` files.  Can you find a few other synth files and make sure we're doing that here?

File: README.md (hunk 1 above)
Comment (telpirion) [reply README.md]: The quickstart isn't deleted, it's just not very robust.

File: synth.py (hunk 2 above)
Comment (telpirion) [reply synth.py]: Removed v1beta1. It sounds like it won't ever become public, so we can remove it.

File: synth.py (hunk 3 above)
Comment (telpirion) [reply synth.py]: Done

File: README.md (hunk 1 above)
Comment (JustinBeckwith) [reply README.md]: Right - but we'd still like to make sure we have one.  Goal is to have a copy/paste snippet on the README that at least helps get folks started :)

File: README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@
 
 
 
-Cloud Document API client for Node.js
+Documentai client for Node.js
```
Comment (JustinBeckwith) [inline README.md]: That don't look right - I can't remember what populates this though 😆

Comment (telpirion) [reply README.md]: Changed.

Comment (telpirion) [reply README.md]: I think that it's populated by the `description` field in the package.json file

**Pull request merged**